### ISSUE TITLE
CO2 budget line and button

### ISSUE THREAD

**Issue by sarahmadeleine** [pledge4future/WePledge#168]
Change the color from green to light blue (#4493c6)

**Comment by veitu** [comment]
the carbon budgets have been calculated. We have two different carbon budgets: One for Germany and one for all other countries. The carbon budgets are [here](https://docs.google.com/spreadsheets/d/1yl8opu0hrcsSdosuI_efW1baQekPxNfKEcXNgtle_PA/edit?usp=sharing)

**Comment by codingfabi** [comment]
@redfrexx currently, the carbon budget is just implemented as a constant into the frontend. Should we keep it this way or can we send it via the backend? 

It could be hard to distinguish between the german budget and the budget of the other countries just from the frontend. I added a backend label but we should first discuss how to implement this properly.

**Comment by redfrexx** [comment]
yes, I also think we should implement a new endpoint for this. I've created a new issue for this #251.
